packages/components/src/navigator/navigator/component.tsx

```diff
@@ -273,41 +273,31 @@ function UnconnectedNavigatorProvider(
 }
 
 /**
- * The `NavigatorProvider` component allows rendering nested views/panels/menus
- * (via the `NavigatorScreen` component and navigate between these different
- * view (via the `NavigatorButton` and `NavigatorBackButton` components or the
+ * The `Navigator` component allows rendering nested views/panels/menus
```

Comment:
The top level `Navigator` component (previously named `NavigatorProvider`) is the only component for which I've kept a JSDoc for the internal implementation, because apparently Storybook uses it when displaying docs 🤷  Not sure why it doesn't use the one defined in `packages/components/src/navigator/index.ts`

Reply:
it seems like the `ts` extension was the problem — changing it to `.tsx` seems to allow Storybook to pick the JSDoc up 😅  https://github.com/WordPress/gutenberg/pull/64613/commits/85471a9224994d8977221fc4915a159622c3f307


cc @mirka